tests/appsec/waf/test_rules.py

```diff
@@ -242,7 +245,7 @@ def test_sqli(self):
         interfaces.library.assert_waf_attack(r, rules.sql_injection.crs_942_160)
 
         r = self.weblog_get("/waf", cookies={"value": "/*!*/"})
-        interfaces.library.assert_waf_attack(r, rules.sql_injection)
+        interfaces.library.assert_waf_attack(r, rules.sql_injection.crs_942_100)
```

Comment:
```suggestion
        interfaces.library.assert_waf_attack(r, rules.sql_injection)
```

Several rules reports this pattern, the WAF is not deterministic here => validate rule type, eg `rules.sql_injection`

Reply:
When I tried it failed if I didn't specify the rule though.

Reply:
Do you have the log output of this fail ?